src/api/types.ts

```diff
@@ -170,8 +178,8 @@ export interface ApplicableRuleOutcome extends RuleOutcomeBase {
 
 export interface NotApplicableRuleOutcome extends RuleOutcomeBase {
   endDate?: string;
-  requestedDate: string;
-  approvedDate: string;
+  requestedDate?: string;
+  approvedDate?: string;
```

Comment:
When would these be null?

Reply:
We're using the not applicable outcome for rule filters as well as rule exemptions, so these don't have any request/approval process and would be null in this case

Reply:
Ah, good to know!